Makefile

```diff
@@ -174,15 +174,15 @@ ifeq ($(OSFLAG), LINUX)
 	sed -i 's|\(image:\).*|\1 $(DOCKER_REGISTRY):$(GITCOMMIT)|g' deploy/namespace-init.yaml
 ifeq ($(KUBECTL_CONTEXT),minikube)
 	sed -i 's|\(imagePullPolicy\): IfNotPresent|\1: Never|g' deploy/namespace-init.yaml
-	sed -i 's|\(args:\).*|\1\ ["--minikube"\]|' deploy/namespace-init.yaml
+	sed -i 's|\(args:\).*|\1\ ["--hostname=$(shell minikube ip)", "--port=NodePort"\]|' deploy/namespace-init.yaml
```

Comment:
would it be better to get kube-context instead of querying for the IP. referencing the kubeconfig thorugh the kube-context would be easier and delegating the responsibility to kubectl would make it easier for the user to use their cluster.
```
kubectl config current-context
``` 
the above command picks up the current context and passing that context as the default env for the jenkins-operator to be tested on makes more sense IMO. @tomaszsek WDYT ?